src/icalendar/tools.py

```diff
@@ -22,8 +39,15 @@ def uid(host_name="example.com", unique=""):
             datetime-uniquevalue@host.
         Like:
             20050105T225746Z-HKtJMqUgdO0jDUwm@example.com
+
+        Use https://docs.python.org/3/library/uuid.html#uuid.uuid5 instead.
```

Comment:
I can't make a suggestion for the entire docstring in the GitHub UI, so this one will need to be applied manually.

```suggestion
"""Generates a unique ID consisting of ``datetime-uniquevalue@host``.
   
   For example:
       
       20050105T225746Z-HKtJMqUgdO0jDUwm@example.com
   
   .. deprecated:: 6.3.0

        Use the Python standard library's :function:`uuid4` instead.
```